### PR TITLE
MM-44829 - try free dont start trial directly from pricing modal

### PR DESCRIPTION
**Pull request by pvev**
#### Summary
This PR adds changes to the pricing modal so the try free button starts the trial directly instead of opening the learn more trial modal

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-44829

#### Related Pull Requests
n/a

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Review by julmondragon — APPROVED** [review]
LGTM!

File: components/pricing_modal/content.tsx [hunk 1]
```diff
@@ -117,12 +119,16 @@ function Card(props: CardProps) {
                     {props.planExtraInformation && props.planExtraInformation}
                 </div>
                 <div>
-                    <button
-                        id={props.id + '_action'}
-                        className={`plan_action_btn ${props.buttonDetails.disabled ? ButtonCustomiserClasses.grayed : props.buttonDetails.customClass}`}
-                        disabled={props.buttonDetails.disabled}
-                        onClick={props.buttonDetails.action}
-                    >{props.buttonDetails.text}</button>
+                    {props.customButtonDetails ? props.customButtonDetails : (
```
Comment (AGMETEOR) [inline components/pricing_modal/content.tsx:122]: Maybe this should fallback to the buttonDetails (which would have cta to contact sales)?

**Review by stevemudie — APPROVED** [review]
Looks good @pvev!

**Pull request merged**